langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/LangLibXMLTest.java

```diff
@@ -187,6 +187,12 @@ public void testCreateComment() {
         assertEquals((returns[0]).stringValue(), "<!--This text should be wraped in xml comment-->");
     }
 
+    @Test
+    public void testCreateText() {
+        BValue[] returns = BRunUtil.invoke(compileResult, "testCreateText");
+        assertEquals((returns[0]).stringValue(), "This is xml text");
```

Comment:
Shall we just invoke the function here and write the assertion in Ballerina itself?

Reply:
All the assertions in this class is done in the class itself, so that is why I did the assertion of testCreateText() here. Is it okay to only change this.

Reply:
Yes, that's OK. We can make sure that whatever new code we add does this, and when we're eventually migrating all the assertions to Ballerina, we only have to fix the old ones.